.swiftlint.yml

```diff
@@ -1,110 +1,106 @@
-# Compiled with swiftlint 0.13.0 rules
+included: # paths to include during linting.
+  - Source
 
-disabled_rules:
-  - conditional_returns_on_newline
-  - cyclomatic_complexity
-  - missing_docs
-  # opening_brace is disabled because it is triggering on the case where a
-  # block parameter is declared on a newline. This conflicts with our style:
-  # "For functions with many argument, specify each arg on new line and )
-  #  on final line"
-  - opening_brace
+excluded: # paths to ignore during linting. Takes precedence over `included`.
+  - Pods
+  - Carthage
+  - UnitTests
+
+reporter: "xcode" # reporter type (xcode, json, csv, checkstyle, junit)
+
+## Rules
+
+# enabled_rules_for_documentation:
+#   - attributes
+#   - class_delegate_protocol
+#   - closing_brace
+#   - closure_end_indentation
+#   - closure_parameter_position
+#   - closure_spacing
+#   - colon
+#   - comma
+#   - compiler_protocol_init
+#   - control_statement
+#   - custom_rules
+#   - discarded_notification_center_observer
+#   - dynamic_inline
+#   - empty_parameters
+#   - empty_parentheses_with_trailing_closure
+#   - first_where
+#   - force_cast
+#   - force_try
+#   - force_unwrapping
+#   - generic_type_name
+#   - identifier
+#   - identifier_name
+#   - implicit_getter
+#   - implicitly_unwrapped_optional
+#   - leading_whitespace
+#   - legacy_cggeometry_functions
+#   - legacy_constant
+#   - legacy_constructor
+#   - legacy_nsgeometry_functions
+#   - mark
+#   - missing_docs
+#   - nesting
+#   - object_literal
+#   - opening_brace
+#   - operator_usage_whitespace
+#   - operator_whitespace
+#   - private_outlet
+#   - private_unit_test
+#   - redundant_discardable_let
+#   - redundant_nil_coalescing
+#   - redundant_optional_initialization
+#   - redundant_string_enum_value
+#   - return_arrow_whitespace
+#   - statement_position
+#   - syntactic_sugar
+#   - trailing_comma
+#   - trailing_newline
+#   - trailing_semicolon
+#   - trailing_whitespace
+#   - type_body_length
+#   - type_name
+#   - unused_closure_parameter
+#   - unused_enumerated
+#   - unused_optional_binding
+#   - valid_docs
+#   - valid_ibinspectable
+#   - vertical_parameter_alignment
+#   - vertical_whitespace
+#   - weak_delegate
 
-  - todo
-  - variable_name
 opt_in_rules:
-  - closure_spacing
-  - empty_count
   - explicit_init
-  - force_unwrapping
+  - fatal_error_message
+  - number_separator
   - overridden_super_call
-  - private_outlet
+  - prohibited_super_call
+  - redundant_void_return
+  - sorted_imports
+
+disabled_rules:
+  - conditional_returns_on_newline
+  - cyclomatic_complexity
+  - empty_count
+  - explicit_type_interface
+  - file_header
+  - for_where
+  - function_body_length
+  - function_parameter_count
+  - large_tuple
+  - line_length
```

Comment:
The first three are code smells.

The last one I personally like a `line_length` of 140 because it allows me on the larger monitors to view two panes side by side easily.